src/metatrain/experimental/alchemical_model/tests/test_functionality.py

```diff
@@ -20,8 +20,13 @@ def test_prediction_subset_elements():
 
     model = AlchemicalModel(MODEL_HYPERS, dataset_info)
 
-    system = ase.Atoms("O2", positions=[[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]])
-    system = systems_to_torch(system)
+    system = System(
+        types=torch.tensor([6, 6]),
+        positions=torch.tensor(
+            [[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]], dtype=torch.get_default_dtype()
```

Comment:
The dtype should be fine without the extra parameter. You are already providing floats.

Reply:
The issue is float vs double

Reply:
But do to the test fail if you remove the dtype?

Reply:
To my surprise, they seem to work (see latest commit)